event_types.go

```diff
@@ -579,10 +579,11 @@ type MergeEvent struct {
 	} `json:"assignee"`
 	Labels  []Label `json:"labels"`
 	Changes struct {
-		AssigneeID struct {
-			Previous int `json:"previous"`
-			Current  int `json:"current"`
-		} `json:"assignee_id"`
+		Assignees []struct {
+			Name      string `json:"name"`
+			Username  string `json:"username"`
+			AvatarURL string `json:"avatar_url"`
+		} `json:"assignees"`
```

Comment:
I don't see these changes in the docs. Could you point me towards a location (in the docs or the source code of Gitlab) or add some API responses that show that this is in fact changed? Thanks!